### PR TITLE
Add --no-verify-tls flag to chaos validate to disable TLS certificate…

### PR DESCRIPTION
**Pull request by dmartin35**
… validation, when using experiment URL over self-signed endpoint

Signed-off-by: David Martin <david@chaosiq.io>

### CODE REVIEW

**Comment by dmartin35** [comment]
see related https://github.com/chaostoolkit/chaostoolkit-lib/issues/163

**Comment by Lawouach** [comment]
Quick question, shouldn't this be a flag on CTK itself?

**Comment by dmartin35** [comment]
> Quick question, shouldn't this be a flag on CTK itself?

there is another PR on the CTK repo, for `chaos run` and `chaos validate` commands

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 [Unreleased]: https://github.com/chaosiq/chaosiq-cloud/compare/0.10.0...HEAD
 
+### Added
+
+-   New `--no-verify-tls` flag to `chaos validate` command
```
Comment (Lawouach) [inline CHANGELOG.md:9]: You meant chaos verify :)

Comment (dmartin35) [reply CHANGELOG.md]: yep good catch

**Comment by dmartin35** [comment]
> minor mistake
> 
> should we release chaostoolkit-lib first and make the new version mandatory for this PR too?

agreed to pin mandatory ctk-lib with an updated version.
there might be other PR you want to include as well before releasing new ctk-lib version .
you probably don't have the time right now though

**Comment by Lawouach** [comment]
actually we can have a release today

**Pull request merged**